go/client/cmd_compat.go

```diff
@@ -10,26 +10,6 @@ import (
 
 // compatibility with node client commands:
 
-func NewCmdCompatEncrypt(cl *libcmdline.CommandLine) cli.Command {
```

Comment:
We should give a nice UX if people use `pgp decrypt` or `decrypt` incorrectly. We can do some basic string matching on the first 50 bytes of the message to let them know what to do.